flink-end-to-end-tests/test-scripts/test_resume_savepoint.sh

```diff
@@ -33,8 +33,8 @@ else
   NUM_SLOTS=$NEW_DOP
 fi
 
-STATE_BACKEND_TYPE=${STATE_BACKEND_TYPE:-file}
-STATE_BACKEND_FILE_ASYNC=${STATE_BACKEND_FILE_ASYNC:-true}
+STATE_BACKEND_TYPE=${3:-file}
+STATE_BACKEND_FILE_ASYNC=${4:-true}
```

Comment:
We should also update the usage message at the beginning of this file to reflect these extra parameters.

Reply:
will do 👍